src/Indigo/Contracts/HomebaseLite/Optimizer.hs

```diff
@@ -1,8 +1,15 @@
 -- SPDX-FileCopyrightText: 2022 Tezos Commons
 -- SPDX-License-Identifier: LicenseRef-MIT-TC
 
+-- NOTE this pragmas.
+-- We disable some warnings for the sake of speed up.
+-- Write code with care.
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
+{-# OPTIONS_GHC -Wno-overlapping-patterns #-}
```

Comment:
I'm assuming that the speed up is in compilation times, in which case I'd remove these.

Reply:
The difference is actually rather substantial, as in with this pragmas it compiles in seconds, while without it takes up to a minute and change. The code here is woefully unsafe anyway, so I don't think this makes much of a difference.

Reply:
OK, I see, in that case I don't mind keeping it :ok_hand: